src/tests/ftest/dfuse/bash_dcache.py

```diff
@@ -0,0 +1,82 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import stat
+
+from dfuse_test_base import DfuseTestBase
+from run_utils import run_remote
+
+SCRIPT = """#!/bin/bash
+
+#!/bin/bash
+
+dir_name="dir00"
+file_name="file00"
+
+rm -rf ${dir_name}
+mkdir ${dir_name}
+echo "Hello" > ${dir_name}/${file_name}
+rm -rf ${dir_name}
+
+mkdir ${dir_name}
+echo "Hello" > ${dir_name}/${file_name}
```

Comment:
Write to a new file, but in the cached, deleted dir.  It's interesting what this would do...  This should probably not also write Hello as you won't know which file you're catting.